balena/models/tag.py

```diff
@@ -392,27 +396,30 @@ def set(self, release_id, tag_key, value):
             OK: if tag exists.
 
         Examples:
-            >>> balena.models.tag.release.set('465307', 'releaseTag1', 'Python SDK')
+            >>> balena.models.tag.release.set(465307, 'releaseTag1', 'Python SDK')
             {u'release': {u'__deferred': {u'uri': u'/balena/release(465307)'}, u'__id': 465307}, u'tag_key': u'releaseTag1', u'id': 135, u'value': u'Python SDK', u'__metadata': {u'type': u'', u'uri': u'/balena/release_tag(135)'}}
-            >>> balena.models.tag.release.set('465307', 'releaseTag1', 'Python SDK 1')
+            >>> balena.models.tag.release.set(465307, 'releaseTag1', 'Python SDK 1')
             OK
 
         """
 
+        release_id = self.release.get(commit_or_id)['id']
+
         return super(ReleaseTag, self).set(release_id, tag_key, value)
 
-    def remove(self, release_id, tag_key):
+    def remove(self, commit_or_id, tag_key):
         """
         Remove a release tag.
 
         Args:
-            release_id (str): release id.
+            commit_or_id: release commit (str) or id (int).
             tag_key (str): tag key.
 
         Examples:
-            >>> balena.models.tag.release.remove('135', 'releaseTag1')
+            >>> balena.models.tag.release.remove(135, 'releaseTag1')
```

Comment:
NIT: Let's change this to `465307` so that it's consistent with the above examples.